### DISCUSSION TITLE
How to add a button on top of an image?

### DISCUSSION THREAD

**Discussion by sukanya-joshi** [h2oai/wave#2312] [Q&A]
Hi! I was wondering if there was a way on Wave to add a button on top of an image? I am currently trying to create an interactive map where the underlying image is a map and the city names are buttons that a user can click on to get more info (it will basically redirect the user to a new card). Thank you so much for the help!!

**Comment by mturoci** [comment]
Hey @sukanya-joshi! Wave currently doesn't provide such mechanism, but should be doable relatively easily with absolute CSS positioning and custom Javascript.
